v2/pkg/commands/build/build.go

```diff
@@ -151,6 +165,51 @@ func Build(options *Options) (string, error) {
 	return compileBinary, nil
 }
 
+func generateBindings(options *Options) error {
+	options.Logger.Print("  - Generating obfuscated bindings: ")
+	defer func() {
+		options.Logger.Println("Done.")
+	}()
+	// Run go mod tidy to ensure we're up-to-date
+	stdout, stderr, err := shell.RunCommand(".", "go", "mod", "tidy")
```

Comment:
We normally use `compat=1.17` for mod tidy, should we do this here too?
```suggestion
	stdout, stderr, err := shell.RunCommand(".", "go", "mod", "tidy", "-compat=1.17")
```